deploy/kubernetes-1.12/pmem-csi.yaml

```diff
@@ -94,7 +94,7 @@ spec:
       - name: external-provisioner
         imagePullPolicy: Always
         image: quay.io/k8scsi/csi-provisioner:v0.4.1
-        args: [ "--v=5", "--provisioner=pmem-csi", "--csi-address=$(ADDRESS)" ]
+        args: [ "--v=5", "--provisioner=pmem-csi", "--csi-address=$(ADDRESS)", "--feature-gates=Topology=true"]
```

Comment:
should there be space near end of line? (likely just style issue, not functional)

Reply:
Yes, i agree. Done